DECISION-MAKING.md

```diff
@@ -31,6 +31,7 @@ BIDS governance.
 | Franklin Feingold ([@franklin-feingold](https://github.com/franklin-feingold)) | 5h/week         | Community development      |
 | Taylor Salo ([@tsalo](https://github.com/tsalo))                               | 3h/week         | MRI                        |
 | Remi Gau ([@Remi-Gau](https://github.com/Remi-Gau))                            | 3h/week         | Community development, MRI |
+| Anthony Galassi  ([@bendhouseart](https://github.com/bendhouseart))            | 3h/week         | PET, ???                   |
```

Comment:
@bendhouseart I will let you edit the details of that line. :smile:

Reply:
@Remi-Gau PET is fine for now, can always update. If you think outreach counts as Community development then chuck that in there too.

Reply:
Done. Feel free to press the big green button :arrow_down: .